### ISSUE TITLE
Dock & Panel only  auto hide if there is an active window/app on that display/screen

### ISSUE THREAD

**Issue by Matt-1-2-3** [pop-os/cosmic-epoch#626]
If you set the dock and panel to auto hide, they will ONLY auto hide if you have an active app/window on that display. Auto hide should mean auto hide always.

I have 2 displays, an HDMI and display port. If I open Firefox on the HDMI the panel and dock will hide on the HDMI but not on display port. If I drag Firefox over to the Display port monitor then on the HDMI monitor the dock and panel re-appear on the HDMI. On the DP monitor both panel and dock hide when Firefox moves there.

**Comment by soloturn** [comment]
why is this behavior not good?

**Comment by Matt-1-2-3** [comment]
Maybe I don't want to see them unless I hover my mouse in that area? Maybe I don't want static elements on my OLED screen for burn in reasons? Etc... 

 Auto hide in every other desktop means hide always, but in cosmic it means auto hide only if something else is on the screen.

**Comment by GetPsyched** [comment]
I disagree here, I prefer having the current behaviour. But since configuration is all about opinions, I would say this being a configuration option would be nice, as long as it can be implemented reasonably without hurting the user experience.

**Comment by dan-kez** [comment]
I would also love to toss my hat in the ring for this feature. Oled burn in is a concern for me and I’d like to have a pure black screen if I want to. Both this and the lack of a super +D (hide all windows) button have been a bit of a surprise to me coming to pop os
